lib/features/recurring_donations/overview/widgets/recurring_donation_item.dart

```diff
@@ -0,0 +1,199 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app/core/enums/country.dart';
+import 'package:givt_app/features/auth/cubit/auth_cubit.dart';
+import 'package:givt_app/features/recurring_donations/overview/models/recurring_donation.dart';
+import 'package:givt_app/utils/app_theme.dart';
+import 'package:intl/intl.dart';
+
+class RecurringDonationItem extends StatefulWidget {
+  const RecurringDonationItem({
+    required this.recurringDonation,
+    required this.isExtended,
+    required this.onTap,
+    required this.onCancel,
+    super.key,
+  });
+
+  final RecurringDonation recurringDonation;
+  final bool isExtended;
+  final void Function() onTap;
+  final void Function() onCancel;
+  @override
+  State<RecurringDonationItem> createState() => _RecurringDonationItemState();
+}
+
+class _RecurringDonationItemState extends State<RecurringDonationItem> {
+  //TODO: POEditor
+  final frequencies = ['week', 'month', 'quarter', 'half year', 'year'];
```

Comment:
Probably they are already in POEditor

Reply:
perhaps you are right, I just separated all POEditor related efforts in separate subtask for now